### PR TITLE
nodes to sense membership changes in the meta cluster

### PR DESCRIPTION
**Pull request by guojidan**
# Required checklist
- [ ] Sample config files updated (`config`,`meta/config` and `default config`) 
- [ ] If there are user-facing changes, the documentation needs to be updated prior to approving the PR( [Link]() )
- [ ] If there are any breaking changes to public APIs, please add the `api change` label.
- [x] Signed [CLA](https://cla-assistant.io/cnosdb/cnosdb) (if not already signed)

# Which issue does this PR close?

[//]: # (We generally require a GitHub issue to be filed for all bug fixes and enhancements and this helps us generate change logs for our releases. You can link an issue to this PR using the GitHub syntax. For -- example `Closes #123` indicates that this PR will close issue #123.)

Related #1633 .

# Rationale for this change

[//]: # (Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
 Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.)

# Are there any user-facing changes?

[//]: # (There is no need to duplicate the description in the issue here, but it is sometimes worth providing a summary of the individual changes in this PR.)

### CODE REVIEW

**Comment by guojidan** [comment]
@roseboy-liu can you review this pr : )

**Review by roseboy-liu — COMMENTED** [review]
Have you ever test for your feature？

**Comment by roseboy-liu** [comment]
Think about this case ： If the requested meta node becomes network isolated, will it result in constantly unable to obtain an available addr?

File: meta/src/client.rs [hunk 1]
```diff
@@ -52,7 +52,10 @@ impl MetaHttpClient {
         })?
     }
 
-    pub async fn watch<T>(&self, req: &(String, String, HashSet<String>, u64)) -> MetaResult<T>
+    pub async fn watch<T>(
+        &self,
+        req: &(String, String, HashSet<String>, u64, Vec<String>),
```
Comment (bartliu827) [inline meta/src/client.rs:57]: why add Vec<String> paramter?

**Comment by guojidan** [comment]
> Would it be simpler and less intrusive to start a timer in AdminMeta, periodically synchronize Members, and update them in MetaClient without affecting existing code?

**Comment by guojidan** [comment]
> Would it be simpler and less intrusive to start a timer in AdminMeta, periodically synchronize Members, and update them in MetaClient without affecting existing code?

Let me re implement it

**Pull request closed without merging**